article/_posts/2019-05-30-data-fusion-remote-sensing.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: review
+title:  "Multisource and Multitemporal Data Fusion in Remote Sensing"
+tags:   deep-learning segmentation remote-sensing CNN hyperspectral multi-task-learning point-cloud
+author: Charles Authier
+pdf:  https://arxiv.org/pdf/1812.08287.pdf
+cite:
+    authors: "Pedram Ghamisi, Behnood Rasti, Naoto Yokoya, Qunming Wang, Bernhard Hofle, Lorenzo Bruzzone, Francesca Bovolo, Mingmin Chi, Katharina Anders, Richard Gloaguen, Peter M. Atkinson, Jon Atli Benediktsson"
+    title:   "Multisource and Multitemporal Data Fusion in Remote Sensing"
+    venue:   "arXiv preprint arXiv:1812.08287. 2018 Dec 19."
+---
+
+*KEYS WORDS: light detection and ranging (LiDAR), terrestrial laser scanning (TLS), synthetic aperture radar (SAR), Gravity Recovery And Climate Experiment (GRACE), Medium Resolution Imaging Spectrometer (MERIS), electromagnetic spectrum (EMS) *
```

Comment:
If these are important for the review, I'd use a section or bold syntax for *keywords*, otherwise the rendering looks a little bit cluttered.

Reply:
Done in 8b8f8b3.